SU2_CFD/src/SU2_CFD.cpp

```diff
@@ -39,24 +39,26 @@
 using namespace std;
 
 int main(int argc, char *argv[]) {
-  
-  unsigned short nZone;
+
   char config_file_name[MAX_STRING_SIZE];
-  bool fsi, turbo;
   bool dry_run = false;
-  std::string filename = "default";
-  
+  int num_threads = omp_get_max_threads();
+  std::string filename = "default.cfg";
+
   /*--- Command line parsing ---*/
-  
+
   CLI::App app{"SU2 v7.0.0 \"Blackbird\", The Open-Source CFD Code"};
-  app.add_flag("-d,--dryrun", dry_run, "Enable dry run mode.\n" 
+  app.add_flag("-d,--dryrun", dry_run, "Enable dry run mode.\n"
                                        "Only execute preprocessing steps using a dummy geometry.");
+  app.add_option("-t,--threads", num_threads, "Number of OpenMP threads per MPI rank.");
```

Comment:
The number of threads per rank can now be set via this command line option, it defaults to whatever is defined by the environment.